File: TypeScript/Codewars/src/resolutions/Highest-and-Lowest.ts
export class Kata {
	highAndLow(numbers: string): string {
		const numbersArray = numbers.split(" ").map(Number);

		return `${Math.max(...numbersArray)} ${Math.min(...numbersArray)}`;
	}
}

const t = new Kata();

console.log(t.highAndLow("42"));

File: Csharp/CodeWars/src/Completed/HighestAndLowest.cs
namespace CodeWars.Completed;

public class HighestAndLowest
{
    public static string HighAndLow(string numbers)
    {
        int[] values = [.. numbers.Split(" ").Select(int.Parse)];
        return $"{values.Max()} {values.Min()}";
    }
}
